packages/datadog-plugin-pg/test/index.spec.js

```diff
@@ -150,7 +148,6 @@ describe('Plugin', () => {
               expect(traces[0][0].meta).to.have.property(ERROR_MESSAGE, error.message)
               expect(traces[0][0].meta).to.have.property(ERROR_STACK, error.stack)
               expect(traces[0][0].meta).to.have.property('component', 'pg')
-              expect(traces[0][0].metrics).to.have.property('network.destination.port', 5432)
```

Comment:
Any way to not remove these tests?

Reply:
Sure thing

Reply:
Weird thing I do not see these lines.... I am going to push again

Reply:
I am not sure how they were removed...

Reply:
Added back 👍